osi_detectedlandmark.proto

```diff
@@ -24,164 +24,219 @@ message DetectedTrafficSign
     //
-    // If SensorData has detected entities and all detections are missing, then 
-    // e.g. the number of sensors can confirm the existence_probability.
-    // 
+    // If \c SensorData has detected entities and all detections are missing,
+    // then e.g. the number of sensors can confirm the
+    // \c #existence_probability.
+    //
     // \note This information can be determined via the detected entities'
     // detections ( \c ...Detection::object_id = 'this detected entity' ) and
     // the sensors (their IDs) to which these detections belong.
     //
-    repeated Identifier sensor_id = 11;    
+    // \note OSI uses singular instead of plural for repeated field names.
+    //
+    repeated Identifier sensor_id = 11;
 
-    // Definition of traffic sign geometries.
     //
-    enum Geometry
+    // \brief Candidates for a detected sign as estimated by the sensor.
+    //
+    message EstimatedMainSign
     {
-        // Geometry of the traffic sign is unknown (must not be used in ground
-        // truth).
-        //
-        GEOMETRY_UNKNOWN = 0;
-        
-        // Geometry of the traffic sign is unspecified (but known).
-        //    
-        GEOMETRY_OTHER = 1;
-        
-        // Traffic sign has a triangular geometry pointing to the top.
-        //    
-        GEOMETRY_TRIANGLE_TOP = 2;
-        
-        //Traffic sign has a triangular geometry pointing down.
-        //    
-        GEOMETRY_TRIANGLE_DOWN = 3;
-        
-        // Traffic sign has a circular geometry.
-        //    
-        GEOMETRY_CIRCLE = 4;
-        
-        // Traffic sign has a square geometry.
-        //    
-        GEOMETRY_SQUARE = 5;
-        
-        // Traffic sign has a diamond geometry.
-        //    
-        GEOMETRY_DIAMOND = 6;
-        
-        // Traffic sign has an octagon geometry.
-        //    
-        GEOMETRY_OCTAGON = 7;
-        
-        // Traffic sign has a geomety of an arrow pointing to the left.
-        //    
-        GEOMETRY_ARROW_LEFT = 8;
-        
-        // Traffic sign has a geomety of an arrow pointing to the right.
-        //    
-        GEOMETRY_ARROW_RIGHT = 9;
-        
-        // Traffic sign has a rectangle geometry. (width is bigger than height)
-        // E.g. one-way
-        //
-        GEOMETRY_RECTANGLE = 10;
-        
-        // Traffic sign that has a pole geometry. (height is bigger than width)
-        // E.g. pole indicating highways exit in xx m.
-        //
-        GEOMETRY_POLE = 11;
-        
-        // Traffic sign that has an oversize rectangle geometry.
-        // E.g. direction plates on highway or city signs.
-        //
-        GEOMETRY_PLATE = 12;
+        // A list of candidates for this traffic sign as estimated by the
+        // sensor.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
+        //
+        repeated CandidateSign candidate = 1;
```

Comment:
candidate_signs,

I have no idea why we lost the complete naming convention now...

Reply:
Do you really think so. See some lines below. If we change consequent, then you must rename geometry --> traffic_sign_geometry...